### ISSUE TITLE
Reload performance insufficient

### ISSUE THREAD

**Issue by alex4200** [BlueBrain/nexus#3306]
**Describe the bug**

When I reload a page in Fusion/Studio it took about 60 seconds (!!!) to reload the page. That is a totally inacceptable bad performance. Even 10 seconds would be bad already! That needs to fixed for usability of working with Fusion.

**What part of Nexus is affected**
- [x] Fusion
- [ ] Delta
- [ ] CLI
- [ ] Documentation

**To Reproduce**
Steps to reproduce the behavior, e.g. for Fusion:
1. Go to 'Studio'
2. Select a studio (in the example "test 62227")
3.reload the page

**Expected behavior**

Page should reload in like 5 seconds. 

**Screenshots**

![Screenshot 2022-05-27 at 13.07.33.png](https://images.zenhubusercontent.com/62714e29b18ff51479a15c78/745b4f05-98a9-4182-8597-ddfe7f4b07d1)

**Additional context**
env: stage Nexus Fusion v.1.8.0-M3-5-g7c346d1e
user: adietz

**Comment by samuel-kerrien** [comment]
Agreed, some queries when run on a SPARQL engine can run a long time. The reason for that is that this engine might not be suited for some kinds of queries (e.g. listing of numerous items). A work around (that is at the discretion of the user building the Studio) is to use Elasticsearch for such kinds of queries. In the future, we would like to leverage Search results (which run in Elasticsearch) to build a Studio Dashboard, both improving performance and making the creation of a Studio dashboard much easier (no need to write a SPARQL or Elasticsearch query).

I am relabelling this an `improvement` rather than a `bug`.
